### PR TITLE
ens ui

### PR DESCRIPTION
**Pull request by enriquesouza**

### CODE REVIEW

File: VultisigApp/VultisigApp/View Models/SendTransaction.swift [hunk 1]
```diff
@@ -20,10 +20,37 @@ class SendTransaction: ObservableObject, Hashable {
...
+            let address = try await factory.resolveENS(ensName: ensName)
+            
+            print("Resolved address \(address)")
+            
+            return address
+            
+        } catch {
+            
+            print("Error to extract the DOMAIN ADDRESS: \(error.localizedDescription)")
+            return ""
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/View Models/SendTransaction.swift:49]: maybe just return "toAddresss" here

**Comment by johnnyluo** [comment]
shouldn't we resolve the ENS name before user click "Next" button on send view?

File: VultisigApp/VultisigApp/View Models/SendTransaction.swift (hunk 1 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/View Models/SendTransaction.swift]: I will return toAddress from now.

**Comment by enriquesouza** [comment]
> shouldn't we resolve the ENS name before user click "Next" button on send view?

In that scenario I will create an Address Service, so we can call it and remove duplication

**Review by Rockindash — APPROVED** [review]

**Pull request merged**